README.md

```diff
@@ -1,5 +1,7 @@
 # styled-components-vs-emotion
-a short doc comparing the popular CSS-in-JS libraries `styled-components` and `emotion`
+A short doc comparing the popular CSS-in-JS libraries `styled-components` and `emotion` as of October 2019. For a more detailed comparison, see:
```

Comment:
```suggestion
_Last updated by @dandv on October 18th, 2019_
A short doc comparing the popular CSS-in-JS libraries `styled-components` and `emotion`. For a more detailed comparison, see:
```

Reply:
I prefer moving this to the start because:
- it'll be the first thing they see
- it gives credit to the last person to update it

Reply:
Haha, no objections to that :)